functions.php

```diff
@@ -313,7 +313,7 @@ function as_get_datetime_object( $date_string = null, $timezone = 'UTC' ) {
 	} elseif ( is_numeric( $date_string ) ) {
 		$date = new ActionScheduler_DateTime( '@' . $date_string, new DateTimeZone( $timezone ) );
 	} else {
-		$date = new ActionScheduler_DateTime( $date_string, new DateTimeZone( $timezone ) );
+		$date = new ActionScheduler_DateTime( $date_string ?? 'now', new DateTimeZone( $timezone ) );
```

Comment:
Note that this breaks under PHP 5.6, which [we still support](https://github.com/woocommerce/action-scheduler/blob/3.4.0/readme.txt#L8).

Reply:
Ouch. Fixed.